### PR TITLE
Add inbound svc setup

### PR DESCRIPTION
**Pull request by agajjala**
- Create VPC Endpoint SVC for inbound NLB
- Create DDB Entry

### CODE REVIEW

File: src/eks_setup_helper.py [hunk 1]
```diff
@@ -82,7 +90,7 @@ def aws_ssm_fetch_parameter(self, parameter_name):
             parameter = self.ssmClient.get_parameter(Name=parameter_name, WithDecryption=True)
         except botocore.exceptions.ClientError as error:
             print("Failed to fetch AWS Systems Manager parameter: %s" % (parameter_name))
-            raise error
+            return ""
```
Comment (kevinwang-sfdc) [inline src/eks_setup_helper.py:93]: Any reason why we return here instead of raising an error?

File: src/eks_setup_helper.py [hunk 2]
```diff
@@ -190,13 +260,79 @@ def inbound_eks_nlb_setup(awsClient, manifest_data):
...
+        Tags = [
+            {"Key": "env_name", "Value": manifest_data["env_name"]},
+            {"Key": "deployment_id", "Value": manifest_data["deployment_id"]},
+            {"Key": "region", "Value": manifest_data["region"]},
+        ]
+        svcID, svcName, svcState = awsClient.aws_create_vpc_endpoint_service_configuration(
+            lb_name=nlb_name.split("-", 1)[0], tags=Tags
+        )
+
+        time.sleep(10)
```
Comment (kevinwang-sfdc) [inline src/eks_setup_helper.py:289]: What's the purpose of the sleep here?

File: src/eks_setup_helper.py [hunk 3]
```diff
@@ -190,13 +260,79 @@ def inbound_eks_nlb_setup(awsClient, manifest_data):
...
+        Filters = [{"Name": "service-name", "Values": [svcName]}, {"Name": "service-state", "Values": ["Available"]}]
+        svcID, svcName, svcState = awsClient.aws_describe_vpc_endpoint_service_configurations(serviceID=svcID, Filters=Filters)
+        print("Created ServiceID: %s, ServiceName: %s, ServiceState: %s" % (svcID, svcName, svcState))
+
+    # Create a VPC EndPoint Connection Notification
+    conn_notif_arn_parameter_key = "/{}-{}/{}/inbound-data-plane/vpce-sns-topic".format(
+        manifest_data["env_name"], manifest_data["region"], manifest_data["deployment_id"]
+    )
+    connNotifARN = awsClient.aws_ssm_fetch_parameter(parameter_name=conn_notif_arn_parameter_key)
+    if connNotifARN == "":
```
Comment (kevinwang-sfdc) [inline src/eks_setup_helper.py:300]: An error message would be good to have here for debugging.

File: src/eks_setup_helper.py [hunk 4]
```diff
@@ -190,13 +260,79 @@ def inbound_eks_nlb_setup(awsClient, manifest_data):
...
+    awsClient.aws_ddb_put_item(inbound_cfg_settings_tbl_name, inbound_ddb_item)
+
 
 def outbound_eks_nlb_setup(awsClient, manifest_data):
     # Fetch SFDCSB.NET Hosted ZOne -ID from AWS SSM
     r53_parameter_key = "/{}-{}/{}/stack_base/r53/sfdcsb".format(
         manifest_data["env_name"], manifest_data["region"], manifest_data["deployment_id"]
     )
     zone_id = awsClient.aws_ssm_fetch_parameter(parameter_name=r53_parameter_key)
+    if zone_id == "":
```
Comment (kevinwang-sfdc) [inline src/eks_setup_helper.py:333]: Same as above.

File: src/eks_setup_helper.py (hunk 2 above)
Comment (agajjala) [reply src/eks_setup_helper.py:294]: SVC Creation takes time, sleeping  for a few seconds before the svc_state becomes available

**Pull request merged**